Example/PaymentSheet Example/PaymentSheet Example/SavedPaymentMethodSheetTestPlayground.swift

```diff
@@ -359,10 +341,9 @@ class SavedPaymentMethodsBackend {
         task.resume()
     }
 
-    func createSetupIntent(customerId: String, completion: @escaping (String?) -> Void) {
-        guard clientSecret == nil else {
-            completion(clientSecret)
-            return
+    func createSetupIntent(customerId: String) async throws -> String {
+        if let clientSecret = clientSecret {
+            return clientSecret
```

Comment:
Did we change this for testing purposes?

Reply:
Weird, removed!